portal-api/api-tests/v1/feedback/feedback.api-test.js

```diff
@@ -38,14 +38,14 @@ describe('/v1/feedback', () => {
   };
 
   describe('POST /v1/feedback', () => {
-    it('401s requests that do not present a valid Authorization token', async () => {
-      const { status } = await as().post({}).to('/v1/feedback');
-      expect(status).toEqual(401);
+    it('200s requests that do not present a valid Authorization token', async () => {
```

Comment:
the status code should be 401. 200 means everything was ok

Reply:
should be 200 as we have removed validation/authorisation to submit feedback

Reply:
in that case, can we remove the test altogether?

Reply:
I'd like to keep it incase we change it again at any point please